imsim/camera.py

```diff
@@ -202,15 +202,16 @@ def __getattr__(self, attr):
         return getattr(self.lsst_ccd, attr)
 
 
-def get_camera(camera):
+_camera_cache = {}
+def get_camera(camera='LsstCam'):
     """
     Return an lsst camera object.
 
     Parameters
     ----------
     camera : str
        The class name of the LSST camera object. Valid names
-       are 'LsstCam', 'LsstComCam', 'Latiss'.
+       are 'LsstCam', 'LsstComCam', 'Latiss'. [default: 'LsstCam']
```

Comment:
Looking at the check for `telescope` above,
```
if telescope != 'LSST':
    raise NotImplementedError("Batoid WCS only valid for telescope='LSST' currently")
```
it occurs to me that we should omit `'Latiss'` from the camera options here and below.

Reply:
OK.  I don't actually know what Latiss is.  I guess that's AuxTel maybe?
I'm fine with removing that for now, but I guess people may want to make simulations of that eventually.  So we should add an issue to add that back in.

Reply:
Ah, nvm I guess.  That's a slitless spectrograph.  Yeah, we're never going to simulate that with ImSim (at least until GalSim Issue [#1018](https://github.com/GalSim-developers/GalSim/issues/1018) is done).  So just removing this is probably right.

Reply:
Right: LSST Atmospheric Transmission Imager and Slitless Spectrograph (LATISS) on the Auxtel.

For the future I wonder if the way to do something like this would be to have dispersive elements in Batoid?